wrappers/first-stage

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh -e
+name=<hw>
+repo="$name-autograder"
+quarter=<quarter>
+org="cse30-$quarter"
```

Comment:
This here might not be too specific. Maybe for generic wrappers we can just specify the repo path?